client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js

```diff
@@ -252,6 +251,8 @@ class DetailedProgramSplit extends React.Component {
       .sortBy("total")
       .value();
 
+    const divHeight = 650 * (graph_ready_data.length / 30) * 2;
```

Comment:
Hmm, where does the 30 and the 2 come from here. Just magic numbers that get the positioning right?

Wish nivo's bar graphs let us explicitly pass in a bar width (or height in this case).

Reply:
Yup these are just magic numbers I tried

Reply:
Haha, hmm. I looked at it a bit and saw some other magic numbers/formulas in that file, so this isn't making it much worse overall.

Just leave a comment about why the magic calculation's there/what it's doing and I'll approve 👍

Reply:
Oh, my page refreshed when I submitted that comment and your commit doing it showed up

Reply:
okay cool! so do u think the comment I made is sufficient?

Reply:
Seems good now!